doc/man3/EVP_PKEY_fromdata.pod

```diff
@@ -28,8 +28,10 @@ EVP_PKEY_fromdata() creates key parameters or a key, given data from
 I<params> and a context that's been initialized with
 EVP_PKEY_param_fromdata_init() or EVP_PKEY_key_fromdata_init().  The result is
 written to I<*ppkey>. The parameters that can be used for various types of key
-are as described in the "Built-in RSA Import/Export Types" section on the
-L<provider-keymgmt(7)> page.
+are as described by the "Common Parameters" section of the
+L<EVP_PKEY-RSA(7)>, L<EVP_PKEY-DSA(7)>, L<EVP_PKEY-DH(7)>, L<EVP_PKEY-EC(7)>,
```

Comment:
Is it possible for these links to go to the "Common Parameters" sections directly?

Reply:
Er...  this comment doesn't make sense to me, as that would have those pages refer to themselves...

Reply:
I meant could the links here point directly to the "common parameters" sections of the other pages?

Reply:
Ah!  Sure, that can be done.  Gotta do some extra in this case, as a naive edit gets you this awfulness:

```
       EVP_PKEY_fromdata() creates key parameters or a key, given data from
       params and a context that's been initialized with
       EVP_PKEY_param_fromdata_init() or EVP_PKEY_key_fromdata_init().  The
       result is written to *ppkey. The parameters that can be used for
       various types of key are as described by "Common RSA parameters" in
       EVP_PKEY-RSA(7), "Common DSA / DH parameters" in EVP_PKEY-DSA(7),
       "Common DH parameters" in EVP_PKEY-DH(7), "Common EC parameters" in
       EVP_PKEY-EC(7), "Common X25519, X448, ED25519 and ED448 parameters" in
       EVP_PKEY-ED448(7), "Common X25519, X448, ED25519 and ED448 parameters"
       in EVP_PKEY-X25519(7), "Common X25519, X448, ED25519 and ED448
       parameters" in EVP_PKEY-X448(7), and "Common X25519, X448, ED25519 and
       ED448 parameters" in EVP_PKEY-ED25519(7) pages.
```

Reply:
Done

Reply:
My POD skills are lacking...